doc/user/sql-grammar/sql-grammar.bnf

```diff
@@ -16,7 +16,7 @@ array_agg ::=
 as_of ::=
   'AS' 'OF' ( 'AT' 'LEAST' )? timestamp_expression
 begin ::=
-  'BEGIN' ( transaction_mode ( ',' transaction_mode )* )?
+  'BEGIN' ( 'ISOLATION LEVEL' ( isolation_level) | ('READ ONLY')?)
```

Comment:
This isn't correct, you can specify both isolation level and read only. I modeled this page after the postgres page which groups these options together: https://www.postgresql.org/docs/current/sql-begin.html

I think this diff chunk should be reverted.

Reply:
Done